test/unit/org/apache/cassandra/config/ConfigCompatibilityTest.java

```diff
@@ -145,7 +151,7 @@ public void diff_4_1() throws IOException
     @Test
     public void diff_5_0() throws IOException
     {
-        diff(TEST_DIR + "/version=5.0-alpha1.yml", ImmutableSet.<String>builder()
+        diff(TEST_DIR + "/version=5.0.0.yml", ImmutableSet.<String>builder()
```

Comment:
@tiagomlalves so when we go to put this into 5.0.x too, just move it back to 5.0-alpha1.yaml here. Then it trunk we introduce 5.1-alpha1.yaml which will be basically a copy of 5.0-alpha1.yaml.

@michaelsembwever  does this make sense to you?

Reply:
I've reverted this change.